client/app/components/items-list/classes/Sorter.js

```diff
@@ -0,0 +1,71 @@
+import { isString, sortBy } from 'lodash';
+
+const ORDER_BY_REVERSE = '-';
+
+export function compile(field, reverse) {
+  if (!field) {
+    return null;
+  }
+  return reverse ? ORDER_BY_REVERSE + field : field;
+}
+
+export function parse(compiled) {
+  compiled = isString(compiled) ? compiled : '';
+  const reverse = compiled.startsWith(ORDER_BY_REVERSE);
+  if (reverse) {
+    compiled = compiled.substring(1);
+  }
+  const field = compiled !== '' ? compiled : null;
+  return { field, reverse };
+}
+
+export default class Sorter {
+  field = null;
+
+  reverse = false;
```

Comment:
I like this syntax. Must we separate with line break? If it's a lint rule I'd remove it..

Reply:
It's eslint rule, and sometimes it's useful, sometimes not. I think it's better to disable it locally in this file

Reply:
How about `"exceptAfterSingleLine": true`. It'll enforce it on methods (which will most likely be multiline and leave class props alone.

Reply:
Yeah, I think it's perfect :+1: I'll try it here